src/applications/disability-benefits/all-claims/migrations/05-truncate-otherAtRiskHousing.js

```diff
@@ -0,0 +1,10 @@
+import omit from 'platform/utilities/data/omit';
+
+export default savedData => {
+  if (savedData.otherAtRiskHousing) {
+    const newData = omit('otherAtRiskHousing', savedData);
+    newData.otherAtRiskHousing = savedData.otherAtRiskHousing.substring(0, 499);
```

Comment:
do we need to add this to the UI behavior or change the forms system to support this kind of limit?

Reply:
We do! it was actually added here, https://github.com/department-of-veterans-affairs/vets-json-schema/pull/352. Do we need to do anything more than that?

Reply:
nope- I just forgot how the forms system work. ignore me